fix: handle relative assets paths in server `fetch` correctly (#12113)
sveltejs/kit: .changeset/dirty-rivers-tickle.md

---

> "@sveltejs/adapter-cloudflare-workers": minor
> "@sveltejs/adapter-cloudflare": minor
> "@sveltejs/adapter-netlify": minor
> "@sveltejs/kit": minor

Reviewer: ```suggestion
"@sveltejs/adapter-cloudflare-workers": patch
"@sveltejs/adapter-cloudflare": patch
"@sveltejs/adapter-netlify": patch
"@sveltejs/kit": patch
```

Author: Thank you!
Sorry, it's my first contribution, so I wasn't sure and the change seamed _minor_ ;)

Reviewer: No worries! Here's where the concept is taken from: https://semver.org/

---

> fix: fix handling of relative paths in fetch on the server

Reviewer: ```suggestion
fix: correctly handle relative paths when fetching assets on the server
```
